tests/virt_autotest/virt_utils.pm

```diff
@@ -198,4 +217,59 @@ sub clean_up_red_disks {
     diag("Debug info: Disks and File Systems Overview:\n $disks_fs_overview");
 }
 
+sub lpar_cmd {
+    my ($cmd, $mesg, $args) = @_;
+    die 'Command not provided' unless $cmd;
+    my $mesg = shift;
+
+    $args->{ignore_return_code} ||= 0;
+    my $ret = console('svirt')->run_cmd($cmd);
+    if ($ret == 0){
+        diag "Command $mesg on S390X LPAR returned: SUCESS";
+    }
+    unless ($args->{ignore_return_code} || !$ret) {
+        diag "Command $mesg on S390X LPAR returned: FAIL";
+        die 'Find new failure, please check manually';
+    }
+}
+
+sub lpar_cmd_with_retry {
```

Comment:
This is never used in your code?

Reply:
lpar_cmd_with_retry api work for install_package.pm and update_package.pm